jep/1/README.adoc

```diff
@@ -553,15 +553,22 @@ community and discuss important objections or concerns raised
 during discussion.
 
 . **Backwards Compatibility** - All JEPs must include a section describing any incompatibilities and their severity.
-  The JEP must explain how the it proposes to deal with these incompatibilities.
+  The JEP must explain how it proposes to deal with these incompatibilities.
   If there are no backwards compatibility concerns, the section must say that.
-. **Security** - All JEPs must include a section describing any security risks and their severity.
-  The JEP must explain how the it proposes to deal with these risks.
-  If there are no security concerns, the section must say that.
+. **Security** - All JEPs must include a section ddescribing their impact
+  on the security of affected components, and, if any, how these are dealt with.
```

Comment:
'these' and 'any' refers to 'impact', I'm not sure that's grammatical. Not sure how else to phrase it. Alternative, countable, term for impact perhaps?

Reply:
@daniel-beck 
Dude, you wrote the thing.  I applied your suggested change. Now you tell me to change it again?  😕  Okay, I'm changing it again.

Reply:
@daniel-beck This is why I wanted you to submit the PR.